pkg/addons/data/DaemonSet.apps/kube-service-catalog/apiserver.yaml

```diff
@@ -1,10 +1,11 @@
 apiVersion: apps/v1
-kind: Deployment
+kind: DaemonSet
```

Comment:
looks like our api server is moving to a daemonset.  Is this desired?

Reply:
This was done by upstream code. Open for discussion what we want to do as product